tests/test_projn.py

```diff
@@ -1,6 +1,10 @@
 """Test projn.py"""
 import torch
 
+from hypothesis import given
+
```

Comment:
There shouldn't be a blank line here (see the "imports" section of the Khan Academy Python style guide.)